frontend/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use small Alpine Linux image
+FROM node:12-alpine
+
+# Copy all local files into the image.
+COPY . .
+
+# Install npm dependencies
+RUN npm ci --only-production --silent
```

Comment:
If you have this we don't need to copy node_modules in line 5. Consider adding a .dockerignore file that excludes the node_modules folder.

Reply:
Good catch, added to https://github.com/methodpark/digitaleswarten/pull/56/commits/82f3f124e96bd4242de2945e0afd8098ce656b59.